tarantool/connection_pool.py

```diff
@@ -218,36 +217,47 @@ def __init__(self,
         '''
         Initialize connections to the cluster of servers.
 
-        :param list addrs: List of {host: , port:} dictionaries,
-        describing server addresses.
-        :user str Username used to authenticate. User must be able
-        to call box.info function. For example, to give grants to 
-        'guest' user, evaluate
-          box.schema.func.create('box.info')
-          box.schema.user.grant('guest', 'execute', 'function', 'box.info')
-        on Tarantool instances.
+        :param list addrs: List of
+
+            .. code-block:: python
+
+                { host: "str",
+                  port: int,
+                  transport: "str",
+                  ssl_key_file: "str",
```

Comment:
Does it possible we'll need a separate certificates for different instances?

Reply:
I think yes, it may be possible. I'm not aware of such restrictions.